README.md

```diff
@@ -10,7 +10,16 @@ Crystal is a programming language with the following goals:
 * Have compile-time evaluation and generation of code, to avoid boilerplate code.
 * Compile to efficient native code.
 
-## Build
+## Contributing to These Docs
```

Comment:
Discussed this with the team and everyone likes "Language Reference" and "API Reference" :). So let's be explicit about that naming in this PR. We're working on a website redesign, so it's a good time to introduce the change there too.